ui/package.json

```diff
@@ -29,7 +29,7 @@
     "react-hotkeys-hook": "^3.3.1",
     "react-markdown": "^5.0.3",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.1",
+    "react-scripts": "4.0.3",
```

Comment:
Is this the only dependency we need to update ??

Reply:
Should we also check if there are updates for the other packages as well

Reply:
I have updated only packages that has npm audit issue, if we look for other packages then there might be some breaking change in ui

Reply:
Yeah but I think with this patch we should also update the other packages if there is an updated version for it

Reply:
Also we can plan to bump the node and the npm version